onnxruntime/test/providers/test_main.cc

```diff
@@ -43,10 +43,26 @@
 
 std::unique_ptr<Ort::Env> ort_env;
 
-int main(int argc, char** argv) {
+#define TEST_MAIN main
+
+#if defined(__APPLE__)
+  #include <TargetConditionals.h>
+  #if TARGET_IPHONE_SIMULATOR || TARGET_OS_IPHONE
+    void ortenv_setup(){
+      OrtThreadingOptions tpo;
+      ort_env.reset(new Ort::Env(&tpo, ORT_LOGGING_LEVEL_WARNING, "Default"));
+    }
+
```

Comment:
Can we take this out of #ifdef, since TEST_MAIN below can call ortenv_setup also